log/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+use lv2_core::feature::*;
+//use std::ffi::CString;
+use std::fmt;
+use std::os::raw::*; //get all common c_type
+use urid::*;
+
+pub struct EntryClass;
+unsafe impl UriBound for EntryClass {
```

Comment:
Looking at the docs again (http://lv2plug.in/ns/ext/log#Entry), it seems `Entry` is only the superclass of all entry types, and not an entry type itself. If anything, `Entry` should be what the trait is named, this type should probably not exist, and entry types can just be named `Error`, `Warning` without the `Class` suffix

Reply:
I used the class suffix just because some part of the LV2 spec use same name for  "Class", "properties" or "instance" even if there is no issue here.

Reply:
LV2's Classes, Properties and Instances are just RDF data for the manifest (`.ttl`) files, they have nothing to do with the types in C or Rust code. You can just rename them without the class suffix. :slightly_smiling_face: 

`Entry` (or `EntryClass` as you wrote it there) is not actually used in code, so it can be left out entirely as well!

Reply:
Did some cleaning on renaming